diff --git a/node_modules/expo-router/build/getLinkingConfig.js b/node_modules/expo-router/build/getLinkingConfig.js
index b6d4703..9f0f7c4 100644
--- a/node_modules/expo-router/build/getLinkingConfig.js
+++ b/node_modules/expo-router/build/getLinkingConfig.js
@@ -11,6 +11,8 @@ exports.getNavigationConfig = getNavigationConfig;
 function getLinkingConfig(routes) {
     return {
         prefixes: [],
+        // Filter out Appcues SDK deep links for router navigation
+        filter: (url) => !url.startsWith('appcues-16daf46b-3231-4e4a-bb3c-273a4e9100dd'),
         // @ts-expect-error
         config: getNavigationConfig(routes),
         // A custom getInitialURL is used on native to ensure the app always starts at
